src/NuGet.Core/NuGet.Build.Tasks/Strings.resx

```diff
@@ -166,4 +166,7 @@
   <data name="ProjectEvaluationSummary" xml:space="preserve">
     <value>Evaluated {0} project(s) in {1:D2}ms ({2} builds, {3} failures).</value>
   </data>
-</root>
+  <data name="Error_CentralPackageVersions_VersionsNotAllowed" xml:space="preserve">
+    <value>The project is opted-in CentralPackageVersionsManagement and its package references {0} have Version defined when not allowed.</value>
```

Comment:
```suggestion
    <value>The project uses Central Package Versions Management, but the following package references have specified the version outside of the packages.props file: {0}</value>
```

I don't like my suggestion a lot, but I don't like "opted-in". I also find that error messages that have the dynamic part of the error at the end, rather than the middle, are easier to search for.

Reply:
I updated the message.